src/bosh-director/lib/bosh/director/metadata_updater.rb

```diff
@@ -19,6 +19,7 @@ def update_vm_metadata(instance, metadata, factory = CloudFactory.create_with_la
         metadata['deployment'] = instance.deployment.name
         metadata['id'] = instance.uuid
```

Comment:
@dpb587-pivotal from the line you are commenting on I guess you ask for `instance_id` in `update_VM_metadata`? We explicitly left that tag out as its value is in the metadata tag `id`. As we are within the context of the VM/instance, I think we should not "namespace" `id` with `instance`. We are using `instance_id` in DISK metadata as we are in the context of the disk and we need to namespace the id of the instance.

Now you could argue why we used `instance_group` in the context of VM metadata and not just `group`. We actually were not sure there. `instance_group` is what people know from the manifest. I guess they would wonder about what `group` would be.

If the outcome of the discussion will be we need to add `instance` namespace here, then this would also mean `instance_name` and `instance_index`. But I hope we do not need the namespace.